README.md

```diff
@@ -47,6 +47,13 @@ To enable pages on your fork:
 8. Push changes to any branch of your fork and then trigger a manual run of `https://github.com/${yourname}/kroxylicious.github.io/actions/workflows/jekyll-gh-pages.yml`,
    supplying the branch you want to checkout and deploy as a parameter. 
 
+### Building Production HTML
+
+Sometimes it can be useful to build the same static output that is deployed to GitHub Pages. For example, you may need to test some Jekyll
+changes produce the URLs you expect in production.
```

Comment:
I think you should spell out a little more clearly what the difference between `build.sh` and `run.sh` is, in use case terms. And probably move this section before the section about running on a fork (which seems to me like a more advanced thing to want to do)